wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/service/TextDocumentService.java

```diff
@@ -99,8 +101,8 @@ public void configureMethods() {
         dtoToDtoList("references", ReferenceParams.class, LocationDto.class, this::references);
         dtoToDtoList("onTypeFormatting", DocumentOnTypeFormattingParams.class, TextEditDto.class, this::onTypeFormatting);
 
-        dtoToDto("completionItem/resolve", ExtendedCompletionItem.class, ExtendedCompletionItemDto.class, this::completionItemResolve);
-        dtoToDto("documentHighlight", TextDocumentPositionParams.class, DocumentHighlightDto.class, this::documentHighlight);
+        dtoToDto("completionItem/resolve", ExtendedCompletionItem.class, ExtendedCompletionItem.class, this::completionItemResolve);
```

Comment:
Did you test this? Why is it different from the lines above (LocationDto, TextEditDto, etc.)?